src/components/SummaryPage.tsx

```diff
@@ -7,50 +8,196 @@ import { Notes } from './summaryPage/Notes.tsx';
 import { TabPanel } from './summaryPage/TabPanel.tsx';
 import InfoTab from './summaryPage/InfoTab.tsx';
 import Loader from './common/Loader.tsx';
-
-interface DataType {
-  [key: string]: {
-    [key: string]: string | number;
-    notes: string;
-  };
-}
-type LabelsType = {
-  [key: string]: string;
-};
+import {
+  SCKAN_DATABASE_SUMMARY_URL_LATEST,
+  SCKAN_DATABASE_SUMMARY_URL_PREVIOUS,
+  FILES,
+  DATABASE_FILES,
+} from '../settings.ts';
 
 const { primaryPurple600, gray500, white } = vars;
 
-const databaseSummaryURL =
-  'https://raw.githubusercontent.com/MetaCell/sckan-explorer/feature/ESCKAN-28/src/data/database_summary_data.json';
-const databaseSummaryLabelsURL =
-  'https://raw.githubusercontent.com/MetaCell/sckan-explorer/feature/ESCKAN-28/src/data/database_summary_labels.json';
-
 const SummaryPage = () => {
-  const [data, setData] = useState<DataType | null>(null);
-  const [labels, setLabels] = useState<LabelsType | null>(null);
+  const [data, setData] = useState<{ [x: string]: null }>({});
+  const [loaded, setLoaded] = useState(false);
   const [value, setValue] = useState(0);
 
   // @ts-expect-error Explanation: Handling Event properly
   const handleChange = (event: React.SyntheticEvent, newValue: number) => {
     setValue(newValue);
   };
   useEffect(() => {
-    fetch(databaseSummaryURL)
-      .then((response) => response.json())
-      .then((jsonData) => {
-        setData(jsonData);
-      })
-      .catch((error) => console.error('Error fetching data:', error));
+    const dataToPull = {
+      Latest: {
+        [FILES.POPULATION]: null,
+        [FILES.PHENOTYPE]: null,
+        [FILES.SPECIES]: null,
+        [FILES.CATEGORY]: null,
+      },
+      Previous: {
+        [FILES.POPULATION]: null,
+        [FILES.PHENOTYPE]: null,
+        [FILES.SPECIES]: null,
+        [FILES.CATEGORY]: null,
+      },
+    };
+
+    const results = {
+      [FILES.POPULATION]: null,
+      [FILES.PHENOTYPE]: null,
+      [FILES.SPECIES]: null,
+      [FILES.CATEGORY]: null,
+    };
+
+    for (const file in FILES) {
+      const request = new XMLHttpRequest();
+      request.open(
+        'GET',
+        SCKAN_DATABASE_SUMMARY_URL_LATEST + DATABASE_FILES[file],
+        false,
```

Comment:
Note (non-blocking): Synchronous fetching is usually discouraged because it can lead to a poor user experience.